lib/engine/action/program_merger_pass.rb

```diff
@@ -16,18 +16,18 @@ def initialize(entity, corporations_by_round:, options:)
 
       def self.h_to_args(h, game)
         {
-          corporations_by_round: h['corporations_by_round']&.map do |r, v|
-                                   [r, v&.map do |c|
-                                         game.corporation_by_id(c)
-                                       end]
-                                 end.to_h,
+          corporations_by_round: h['corporations_by_round']&.transform_values do |v|
```

Comment:
TIL `Hash#transform_values`, feels like something I should have already known about 😆 

Nice!

Reply:
rubocop docs and [fast-ruby](https://github.com/JuanitoFatas/fast-ruby) are gold mines of knowledge :)